src/lib/TranscriptEditor/TimedTextEditor/WrapperBlock.js

```diff
@@ -54,6 +55,11 @@ class WrapperBlock extends React.Component {
     this.props.blockProps.setEditorNewContentState(newContentState);
   }
 
+  handleTimecodeClick =(e) => {
```

Comment:
spacing (I need to change the linter rule sbecause this is in several places!)

Reply:
oh you mean `= (e) => ` yes, please let's change the linter so that when save it auto lints that way etc..